wp-content/themes/oceanwp-child/functions.php

```diff
@@ -410,4 +411,13 @@ function checkIsFollowing() {
 	$posts = $wpdb->get_results("SELECT DISTINCT userID, postID FROM ggrc_follow_posts WHERE `userID` = '$current_user_id' and `postID` = '$postid' and `isFollowing` = 1");
 
 	return count($posts);
-}
\ No newline at end of file
+}
+
+/* Blog Functions */
+function add_blog_Category($classes) {
+	$blogcat = wp_get_post_terms(get_the_ID(), 'blog-category', ['']);
+	$classes[] = strtolower($blogcat[0]->name);
+
+	return $classes;
+}
+add_filter('body_class', 'add_blog_category');
```

Comment:
Function name should be `add_blog_category` instead of `add_blog_Category` as I can see that is what is passed in the `add_filter` and it will be consistent with our other function names.